src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -468,11 +468,51 @@ function parseBuildGradleFile(gradleFile: string) {
     throw new Error(`Unable to find gradle file "${buildGradlePath}".`);
   }
 
+  return buildGradlePath;
+}
+
+function parseBuildGradleFile(gradleFile: string) {
+  const buildGradlePath: string = getBuildGradleFilePath(gradleFile);
   return g2js.parseFile(buildGradlePath).catch(() => {
     throw new Error(`Unable to parse the "${buildGradlePath}" file. Please ensure it is a well-formed Gradle file.`);
   });
 }
 
+// parse gradle.properties file
+async function parseGradlePropertiesFile(gradleFile: string) {
+  // find gradle.properties file based on gradleFile
+  const buildGradlePath: string = getBuildGradleFilePath(gradleFile);
```

Comment:
I think it's better to remove the dependence of this function on the function of getting the path to the build gradle file.

Reply:
Could you explain why we need this? thank you.

Reply:
Because I think this dependency is redundant. We can safely get the path to the property file without using the path to the build.gradle file, using only the argument of this function.

Reply:
@DmitriyKirakosyan Sorry I can't understand what should I do. 😢

Reply:
Sorry, it looks like I misunderstood part of the code, so you can ignore this comment of mine.